frontend/src/components/guideCarousel/index.js

```diff
@@ -0,0 +1,218 @@
+    leggedRobotImage: robot,
+    title: 'Moves the robot (holo):',
+    activeButtonsDescription: RobotMovementObliquelyHolo,
+    keyboardSettings: {
+      template: 'shift',
+      actions: ['left-forward', 'right-forward', 'right-backward', 'left-backward'],
+      current: 'left-forward',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Roll:',
+    activeButtonsDescription: Roll,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['roll-left', 'roll-right'],
+      current: 'roll-right',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Pitch:',
+    activeButtonsDescription: Pitch,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['pitch-front', 'pitch-back'],
+      current: 'pitch-back',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Robot torsion:',
+    activeButtonsDescription: Torsion,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['torsion-right', 'torsion-left'],
+      current: 'torsion-right',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Robot tilt:',
+    activeButtonsDescription: Tilt,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['tilt-front', 'tilt-back'],
+      current: 'tilt-front',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Robot tilt:',
+    activeButtonsDescription: TiltSide,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['tilt-right', 'tilt-left'],
+      current: 'tilt-right',
+    },
+  },
+  {
+    leggedRobotImage: robot,
+    title: 'Body elevetion:',
+    activeButtonsDescription: BodyElevetion,
+    keyboardSettings: {
+      template: 'normal',
+      actions: ['elevetion-up', 'elevetion-down'],
+      current: 'elevetion-up',
+    },
+  },
+]
+
+
+const CarouselItem = ({ item }) => {
+  return (
+      <Grid container sx={{ display: 'block', marginTop: '15px' }}>
+        <Grid
+          component={'img'}
+          src={item.leggedRobotImage}
+          sx={{
+            display: 'block',
+            maxHeight: '200px',
+            margin: '0 auto 30px',
+          }}
+        />
+        <Grid
+          component='p'
+          sx={{
+            textAlign: 'center',
+            fontSize: '16px',
+            marginBottom: '32px',
+          }}
+        >{item.title}</Grid>
+        <Grid sx={{ textAlign: 'center', marginBottom: '28px' }}>
+          {item.activeButtonsDescription && <item.activeButtonsDescription />}
+        </Grid>
+        <KeyboardTemplate {...item.keyboardSettings} />
+      </Grid>
+  )
+}
+
+const NavButton = ({onClick, className, style, next, prev}) => {
+  return (
+      <Button onClick={onClick} className={className} style={style}>
+          {next && <>Next <ChevronRightIcon/></>}
+          {prev && <><ChevronLeftIcon/> Previous</>}
+      </Button>
+  )
+}
+
+const GuideCarousel = () => {
+  const onChange = (now, prev) => {
+    if(now !== prev) {
+      console.log('start animatino here')
```

Comment:
Do you plan to add something here? + you have a typo in animation

Reply:
type fixed
it's a good place to add logic for animation on slide change

Reply:
yes I do, it's a good place to handle animation logic on slide change